packages/cli/README.md

```diff
@@ -15,47 +15,50 @@ $ npx @lionweb/cli
 
 produces information about which commands are available.
 
-## Extracting essential information from a serialization chunk
+This CLI utility does not perform any explicit validation apart from the file at the given path existing and being valid JSON.
+It does some implicit validation as it can error out on incorrect serializations.
+
+
+## Sorting a serialization chunk
 
-Run the following command to make "extractions" from a serialization chunk (e.g.):
+Run the following command to sort a serialization chunk (e.g.):
```

Comment:
Why sort at all?

Reply:
Mainly for the purpose of being able to compare two chunks for equality. We probably should pull the differ a bit to the fore, instead.